pkgs/development/tools/parsing/tree-sitter/grammar.nix

```diff
@@ -14,6 +15,7 @@
   src,
   location ? null,
   generate ? false,
+  broken ? false,
```

Comment:
seems like it is `brokenTests` rather than the whole grammar ? might be best to add overrides for those grammars in an overrides.nix rather than automate it. Among the advantages is that we dont introduce a new setting `broken`, it would be `doCheck = false` like for every other package, and it encodes it just in overrides.nix rather than the json + nix.